data/json/items/tool_armor.json

```
@@ -2860,6 +2860,110 @@
     "flags": [ "WATERPROOF", "STURDY" ],
     "armor": [ { "encumbrance": 20, "coverage": 95, "covers": [ "head" ] } ]
   },
+  {
+    "id": "helmet_sallet",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "sallet helm" },
+    "description": "A popular german helmet of the late medieval era, the sallet covers the head and eyes.  Activate it to toggle the visor.",
+    "weight": "1800 g",
+    "volume": "2050 ml",
+    "price": 35000,
+    "price_postapoc": 4500,
+    "use_action": { "type": "transform", "msg": "You raise the sallet visor.", "target": "helmet_sallet_raised", "menu_text": "Raise" },
+    "bashing": 8,
+    "material": [ "steel", "leather" ],
+    "symbol": "s",
+    "looks_like": "helmet_barbute",
+    "color": "light_gray",
+    "armor": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 20 },
+      { "covers": [ "eyes" ], "coverage": 100, "encumbrance": 20 }
```

```suggestion
      { "covers": [ "eyes" ], "coverage": 60, "encumbrance": 20 }
```
100% coverage would be a clear transparent material over the eyes with no gaps.